---
string response
Pose waiting_location